src/util/dates.js

```diff
@@ -0,0 +1,15 @@
+export function formatDate(isoDateString) {
```

Comment:
I eliminated the need for moment because:
1. Moment is huge
2. Native JS can handle this use case.

Reply:
moment has been removed